cosmo/serializer.py

```diff
@@ -66,9 +66,16 @@ def render_addresses(self):
         retVal.update(secondaryIPs)
         return retVal
 
+class RouterSerializerConfig:
+    def __init__(self, config = {}):
+        self.config = config
+
+    @property
```

Comment:
For now it's good but maybe we should implement something more generic for the future ? like __getattr__ or something. Not sure if we really need to do this, it's up to you.

Reply:
I need a default somehow.